src/main/java/com/aws/greengrass/clientdevices/auth/ClientDevicesAuthService.java

```diff
@@ -290,8 +294,10 @@ private void updateCAType(Topic topic) {
             uploadCoreDeviceCAs(caCerts);
             updateCACertificateConfig(caCerts);
             updateCaPassphraseConfig(certificateManager.getCaPassPhrase());
+        } catch (CloudServiceInteractionException e) {
+            logger.atError().log(e.getMessage());
```

Comment:
why aren't we logging the actual cause? Why only the message? This can hide valuable info.

Reply:
Nevermind.

Maybe add a comment though that this is logged by uploadCoreDeviceCAs, though I'm not so sure that's the best idea. Maybe that method shouldn't log, and should only throw and let this log it.